tests/zzz_test.py

```diff
@@ -0,0 +1,8 @@
+def test_collect_report(host):
+    host.run('mkdir -p logs')
+    for container, filename in [('foreman', '/var/log/foreman/production.log')]:
+        localfile = filename.replace('/', '_')
+        host.run(f'podman cp {container}:{filename} logs/{container}-{localfile}')
+    host.run('tar caf logs.tar.gz logs/')
+    with open('logs.tar.gz', 'wb') as logstar:
+        logstar.write(host.file('logs.tar.gz').content)
```

Comment:
Crazy? Yes.
Works? Yes.
Did it help? No, for some reason the tests didn't want to fail since I've added this.

Reply:
There are [reporting hooks](https://docs.pytest.org/en/stable/reference/reference.html#reporting-hooks) that can be used. https://docs.pytest.org/en/stable/example/simple.html#post-process-test-reports-failures has an example that looks useful and better than to capture it in a test.

But I think it raises an interesting topic: how do we deal with logging. It's often considered a good practice to log to stdout in containers and then we can simply capture the journal for our services. Should we capture that in another issue?

Reply:
other than https://github.com/theforeman/foreman-quadlet/issues/21? :)

Reply:
Well, I could have checked for issues first.